src/rendertron.ts

```diff
@@ -14,6 +14,7 @@ const CONFIG_PATH = path.resolve(__dirname, '../config.json');
 
 type Config = {
   datastoreCache: boolean;
+  memoryCache: boolean;
```

Comment:
How about `cache: 'datastore'|'memory'|null`? That way it would be clear in the configuration format that the cache choices are mutually exclusive.

Reply:
That's a good idea!